<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoEdit</class>
 <widget class="QMainWindow" name="VideoEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>856</width>
    <height>762</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>856</width>
    <height>762</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>856</width>
    <height>762</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Video Editor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>680</y>
      <width>834</width>
      <height>30</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="comboLabName">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <item>
        <property name="text">
         <string>Hindernis 1 Auffahrt</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Hindernis 1 Abfahrt</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QTimeEdit" name="timeLabTime">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="displayFormat">
        <string>HH:mm:ss</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonLabInsert">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Hinzufügen</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonLabDelete">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Löschen</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonLabEdit">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Bearbeiten...</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="LabelName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>650</y>
      <width>834</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Label &lt;Dateiname&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labPlotName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>834</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Plot &lt;Dateiname&gt;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textPlot">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>467</y>
      <width>834</width>
      <height>171</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Plot Graph&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textVideo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>471</width>
      <height>391</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Video View&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labVideoName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>834</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Video &lt;Dateiname&gt;</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>40</y>
      <width>351</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>856</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatei">
    <property name="title">
     <string>Datei</string>
    </property>
    <addaction name="action_ffnen"/>
    <addaction name="actionSpeichern"/>
    <addaction name="actionSpeichern_unter"/>
   </widget>
   <addaction name="menuDatei"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <action name="action_ffnen">
   <property name="text">
    <string>Öffnen...</string>
   </property>
  </action>
  <action name="actionSpeichern">
   <property name="text">
    <string>Speichern</string>
   </property>
  </action>
  <action name="actionSpeichern_unter">
   <property name="text">
    <string>Speichern unter..</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
